app/templates/components/forms/admin/settings/system/mail-settings/test-email-form.hbs

```diff
@@ -4,13 +4,15 @@
       <i class="icon dropdown"></i>
       {{t 'Send Test Email'}}
     </div>
-    <div class="content four wide field">
```

Comment:
Make changes to this `div` only, there is no need to add an extra `div`.

Reply:
without "ui grid" class how it is possible to make it responsible for both mobile and desktop @kushthedude 
Is there any problem by adding an extra div .

Reply:
Use directly `content 4 wide computer 6 wide mobile field`

Reply:
I have tried content 4 wide computer 16 wide mobile field but without ui grid how it is supposed to work . so i have to add extra div . and for mobile it should be 16 wide mobile field.